### ISSUE TITLE
Text color fails a11y test

### ISSUE THREAD

**Issue by andrewreifman** [makeswift/vibes#210]
https://github.com/makeswift/vibes/blob/63934d11e79ebf9bf69473835ce07315a246938d/apps/web/vibes/soul/components/accordions/index.tsx#L38

We might need to bump this to a darker contrast color for a11y

**Comment by Samantha2233** [comment]
This is resolved in: https://github.com/makeswift/vibes/pull/227/commits/1766b867fe9044df5cd0e6d44b3e5af2709bc3d8
